### DISCUSSION TITLE
Option to "sweep" extra leftover from multiple categories

### DISCUSSION THREAD

**Discussion by bpaulien** [actualbudget/releases#77] [Feature ideas]
I think it would be awesome if we could have an option to sweep the money out of multiple envelopes that still had extra at the end of the month into a savings category, or extra debt category, or whatever category we wanted.  Some of my discretionary categories I manually move money out of at the end of the month, but if I had a "sweep" option, I'd use it to move money more quickly.  

It could possibly even go both ways.  Maybe a category was negative at the end of the month, and some could be swept out of other categories to cover that, and the rest would go to whatever the current savings goal is.

**Comment by ColGra** [comment]
I can see the attraction but by taking automation too far you can become removed from being "hands on" with decisions impacting your budget. Not something I would want but no harm if it is at the users discretion.

**Reply by bpaulien** [reply]
Exactly.  I'm not talking automated.  I'm saying, give me a screen where I can choose what money to move from what category.  I (long long ago) was using the program Mvelopes, and they had a feature like this.  It would display your categories, along with the balance of each.  Then there was a "sweep into" and a "sweep out" column, which you could either manually populate, or allow "full balance".  So you'd either select the category's checkbox,  indicating move everything (in or out, depending on -/+ sign) or you could select a certain amount, and which direction you wanted to move it.

There was also a "sweep into" category that you could select, which would receive any extra.  You had full control, it just made it easier than making a bunch of individual transfers between individual categories at the end of each month.

**Reply by Deguello** [reply]
I don't know. I was "hands on" when I decided how much money I wanted in each category, which categories I wanted to cap, and the cap amount. I don't think it's less "hands on" to simplify sweeping money out of those categories.

Personally, it would be pretty far down on my priority list, but I see why people would like it.

That said, I do like automation here. I wrote a node.js app to send any excess money to a given category. It doesn't save a ton of time but once I get it integrated with the ability to populate my monthly budget amounts, I think it will be pretty slick. Really liking Actual so far.

**Reply by ColGra** [reply]
We shall have to disagree then. Which is why it should be user selectable. It is not even on my bucket list so if it never happens I would not be concerned personally.

**Reply by Deguello** [reply]
Yeah, the app definitely shouldn't move money around without the user's approval.

**Reply by bpaulien** [reply]
Oh, for sure, definitely user selectable.  If you don't want to sweep anything, don't do it.  If you want to, then select the "sweep" option and go to town.
